linkis-web/src/apps/linkis/module/datasourceAccess/index.vue

```diff
@@ -242,6 +242,7 @@ export default {
                 duration: 3,
                 content: "添加成功"
```

Comment:
It is recommended to put it in i18n

Reply:
@mayinrain

Reply:
Thank you for your suggestion! Currently, the access to this page is still blocked, and internationalization optimization will be performed upon the release of future features.